discovery/utils/dataframe_matcher.py

```diff
@@ -72,3 +85,93 @@ def _match_columns(self, reference_column_name, subject_column_name):
 
         # Produce a similarity percentage as an average of all checks
         return round(sum(percentages) / len(percentages), 2)
+
+    @staticmethod
+    def match_column_names():
```

Comment:
This should probably exist as a (non unit) test rather than being part of the library itself, an end user would have no interest in this method.

Reply:
Wait.... _non_-unit test?

Reply:
I'd say that it doesn't make sense as a unit test, as there is no hard pass/fail case
Building it into a unit test would mean having to rebuild the unit test every time you change the methodology of the matcher